vision/segmentation/plane_segmentation.go

```diff
@@ -20,16 +20,18 @@ import (
 var sortPositions bool
 
 // GetPointCloudPositions extracts the positions of the points from the pointcloud into a Vec3 slice.
-func GetPointCloudPositions(cloud pc.PointCloud) []r3.Vector {
+func GetPointCloudPositions(cloud pc.PointCloud) ([]r3.Vector, []pc.Data) {
```

Comment:
This also returns the data now because point cloud access with At() is slower than just grabbing the data from a slice using an index